docs/tutorials/mpv.md

```diff
@@ -399,8 +399,8 @@ sub-ass-override=force
 deband=yes
 ```
 
-!!!
-In this example, the `simulcast` auto profile will automatically apply when you play a release by SubsPlease, Erai-raws, Tsundere-Raws, VARYG, or HorribleSubs.
+!!!warning
+Your auto profile should be placed at the end of your `mpv.conf` in order to prevent conflict.
```

Comment:
profile(s)?

Reply:
Resolved in https://github.com/Snaacky/thewiki/pull/150/commits/feee238e9da3cda5e2dee995a8d6e18f71366fab.